JenkinsfileRT

```diff
@@ -108,11 +108,11 @@ bc1.build_cmds = [
     "pip freeze",
 ]
 bc1.build_cmds = PipInject(env.OVERRIDE_REQUIREMENTS) + bc1.build_cmds
```

Comment:
By leaving the `bc1.build_cmds` uncommented, does that mean it's still going to do the build, but just not actually run the tests? Or is the build not executed until `bc1.test_cmds` is executed?

Reply:
It should only run the build commands on `utils.run()`, so yes in this current config it will build and not test

Reply:
So isn't it a waste of time to create the build and then not run it?  Why not also comment out the build?

Reply:
https://github.com/spacetelescope/jwst/blob/33f102e778fab17f34607c6e2303b5cac88382d0/JenkinsfileRT#L97-L99

I forgot, this is what creates the `macos-stable_deps.txt`, so it is necessary to have here